build.sbt

```diff
@@ -453,4 +453,70 @@ lazy val core = (project in file("modules/core"))
     )
   )
 
+lazy val currencyL0 = (project in file("modules/currency-l0"))
+  .enablePlugins(AshScriptPlugin)
+  .enablePlugins(JavaAppPackaging)
+  .dependsOn(keytool, kernel, shared % "compile->compile;test->test", testShared % Test, dagShared % "compile->compile;test->test", sdk)
+  .settings(
+    name := "tessellation-currency-l0",
+    Defaults.itSettings,
+    scalafixCommonSettings,
+    commonSettings,
+    commonTestSettings,
+    dockerSettings,
+    makeBatScripts := Seq(),
+    libraryDependencies ++= Seq(
+      CompilerPlugin.kindProjector,
+      CompilerPlugin.betterMonadicFor,
+      CompilerPlugin.semanticDB,
+      Libraries.cats,
+      Libraries.catsEffect,
+      Libraries.catsRetry,
+      Libraries.circeCore,
+      Libraries.circeGeneric,
+      Libraries.circeParser,
+      Libraries.circeRefined,
+      Libraries.circeShapes,
+      Libraries.cirisCore,
+      Libraries.cirisEnum,
+      Libraries.cirisRefined,
+      Libraries.derevoCore,
+      Libraries.derevoCats,
+      Libraries.derevoCirce,
+      Libraries.doobieCore,
+      Libraries.doobieHikari,
+      Libraries.doobieH2,
+      Libraries.doobieQuill,
```

Comment:
are all dependencies needed? I'd recommend starting with minimum and extend when required

Reply:
Not really, I just wanted to speed up the initial setup. I will clean it up.